packages/@aws-cdk/aws-lambda-nodejs/lib/esbuild-installation.ts

```diff
@@ -0,0 +1,35 @@
+import { spawnSync } from 'child_process';
+import { getModuleVersion } from './util';
+
+/**
+ * An esbuild installation
+ */
+export abstract class EsbuildInstallation {
+  public static detect(): EsbuildInstallation | undefined {
+    try {
+      try {
+        // Check local version first
+        const version = getModuleVersion('esbuild');
```

Comment:
Why does this have to throw if the module doesn't exist? Can this be `tryGetModuleVersion`?